data/conf/hive-site.xml

```diff
@@ -326,11 +326,6 @@
   <value>false</value>
 </property>
 
-<property>
-  <name>hive.strict.timestamp.conversion</name>
-  <value>false</value>
-</property>
```

Comment:
Test HS2 clusters inherit this hive-site.xml, then this param is stored in where `new HiveConf()` picks up.
In order to make sure that test cases use the param not in hive-site but in session params, I'd like to remove it.